src/registrar/models/utility/generic_helper.py

```diff
@@ -102,34 +102,37 @@ def _handle_new_instance(self):
-            raise ValueError("Cannot update organization_type and generic_org_type simultaneously.")
-        elif not organization_type_changed and (not generic_org_type_changed and not is_election_board_changed):
-            # No changes found
-            if force_update_when_no_are_changes_found:
-                # If we want to force an update anyway, we can treat this record like
-                # its a new one in that we check for "None" values rather than changes.
-                self._handle_new_instance()
-        else:
-            # == Update the linked values == #
-            # Find out which field needs updating
-            organization_type_needs_update = generic_org_type_changed or is_election_board_changed
-            generic_org_type_needs_update = organization_type_changed
-
-            # Update the field
-            self._update_fields(organization_type_needs_update, generic_org_type_needs_update)
+        try:
+            # Instance is already in the database, fetch its current state
+            current_instance = self.sender.objects.get(id=self.instance.id)
+
+            # Check the new and old values
+            generic_org_type_changed = self.instance.generic_org_type != current_instance.generic_org_type
+            is_election_board_changed = self.instance.is_election_board != current_instance.is_election_board
+            organization_type_changed = self.instance.organization_type != current_instance.organization_type
+
+            # == Check for invalid conditions before proceeding == #
+            if organization_type_changed and (generic_org_type_changed or is_election_board_changed):
+                # Since organization type is linked with generic_org_type and election board,
+                # we have to update one or the other, not both.
+                # This will not happen in normal flow as it is not possible otherwise.
+                raise ValueError("Cannot update organization_type and generic_org_type simultaneously.")
+            elif not organization_type_changed and (not generic_org_type_changed and not is_election_board_changed):
+                # No changes found
+                if force_update_when_no_are_changes_found:
+                    # If we want to force an update anyway, we can treat this record like
+                    # its a new one in that we check for "None" values rather than changes.
+                    self._handle_new_instance()
+            else:
+                # == Update the linked values == #
+                # Find out which field needs updating
+                organization_type_needs_update = generic_org_type_changed or is_election_board_changed
+                generic_org_type_needs_update = organization_type_changed
+
+                # Update the field
+                self._update_fields(organization_type_needs_update, generic_org_type_needs_update)
+        except self.sender.DoesNotExist:
+            pass
```

Comment:
**Suggestion optional non-blocking** I would add a comment line maybe above the "pass" just describing that this error only occurs when using the import /export feature.